### PR TITLE
861486 Prepare UG For MAUI Tools/Core Control

### PR DESCRIPTION
**Pull request by surya3655**
**Description** 

Add a UG documentation for the NumericEntry, AvatarView and Chip Control.

### CODE REVIEW

File: MAUI/Chips/Customization.md [hunk 1]
```diff
@@ -1047,6 +1047,48 @@ public MainPage()
 
 ![SfChipGroup with SelectedChipBackgroundColor](images/customization-images/chipgroup_chipbackground_image.png)
 
+## SelectedChipBackground
+
+The `SelectedChipBackground` property customizes the background color of the selected chip.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Chips/Customization.md:1052]: The `SelectedChipBackground` property customizes the background color of the selected chips.

File: MAUI/Chips/Customization.md [hunk 2]
```diff
@@ -1180,6 +1222,48 @@ public MainPage()
 
 N> The default value of SelectedChipTextColor is [`Color.White`].
 
+## SelectedChipBackground
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Chips/Customization.md:1225]: SelectedChipTextColor, Please check the API name

File: MAUI/Chips/Customization.md [hunk 3]
```diff
@@ -1180,6 +1222,48 @@ public MainPage()
...
+
+The `SelectedChipBackground` property customizes the text color of the selected chip.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Chips/Customization.md:1227]: The `SelectedChipTextColor` property customizes the text color of the selected chips.

File: MAUI/NumericEntry/UpDown-Button.md [hunk 4]
```diff
@@ -69,6 +69,141 @@ sfNumericEntry.UpDownPlacementMode = NumericEntryUpDownPlacementMode.Inline;
 
 ![UpDown Placement in .NET MAUI NumericEntry](UpDownButton_images/UpDownButtonPlacement.gif)
 
+## UpDown button color customization
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/NumericEntry/UpDown-Button.md:72]: Change titles like below,

## UpDown button customization

### UpDown button color
### UpDown button template

File: MAUI/Chips/Customization.md (hunk 1 above)
Comment (surya3655) [reply MAUI/Chips/Customization.md:1052]: Changed

File: MAUI/Chips/Customization.md (hunk 3 above)
Comment (surya3655) [reply MAUI/Chips/Customization.md:1227]: Changed

File: MAUI/Chips/Customization.md (hunk 2 above)
Comment (surya3655) [reply MAUI/Chips/Customization.md]: API name is SelectedChipTextColor

File: MAUI/NumericEntry/UpDown-Button.md (hunk 4 above)
Comment (surya3655) [reply MAUI/NumericEntry/UpDown-Button.md]: Changed

**Review by Choza-rajan — APPROVED** [review]

File: MAUI/Avatar-view/customization.md [hunk 5]
```diff
@@ -13,6 +13,50 @@ documentation: ug
 
 The [SfAvatarView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfAvatarView.html?tabs=tabid-1) control provides options to customize the color and size. The control can be customized using the following properties:
 
+## Aspect
+
+The `Aspect` property in [SfAvatarView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfAvatarView.html?tabs=tabid-1) controls how the picture is adjusted to fit the display. By default, its set to `AspectFill`
```
Comment (Chrispineagunja) [inline MAUI/Avatar-view/customization.md:18]: The `Aspect` property in [SfAvatarView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfAvatarView.html?tabs=tabid-1) controls how the picture is adjusted to fit the display. By default, it is set to `AspectFill`.

File: MAUI/NumericEntry/UpDown-Button.md [hunk 6]
```diff
@@ -69,6 +69,143 @@ sfNumericEntry.UpDownPlacementMode = NumericEntryUpDownPlacementMode.Inline;
 
 ![UpDown Placement in .NET MAUI NumericEntry](UpDownButton_images/UpDownButtonPlacement.gif)
 
+# UpDown button customization
+
+## UpDown button color
+
+You can customize the `NumericEntry` control button color by using the `UpDownButtonColor` property.
```
Comment (Chrispineagunja) [inline MAUI/NumericEntry/UpDown-Button.md:76]: Customize the `NumericEntry` control button color by using the `UpDownButtonColor` property.

File: MAUI/NumericEntry/UpDown-Button.md [hunk 7]
```diff
@@ -69,6 +69,143 @@ sfNumericEntry.UpDownPlacementMode = NumericEntryUpDownPlacementMode.Inline;
...
+                 Padding = new Thickness(0, 5, 0, 0),
+                 Rotation = 270,
+                 FontFamily = "FontIcons",
+                 HorizontalOptions = LayoutOptions.Center,
+                 Text = "\ue74a",
+                 TextColor = Colors.Red,
+                 FontSize = 20
+             };
+
+             // Add the Label to the Grid
```
Comment (Chrispineagunja) [inline MAUI/NumericEntry/UpDown-Button.md:193]: // Add a Label to the Grid.

File: MAUI/NumericEntry/UpDown-Button.md [hunk 8]
```diff
@@ -69,6 +69,143 @@ sfNumericEntry.UpDownPlacementMode = NumericEntryUpDownPlacementMode.Inline;
...
+             grid.Children.Add(label);
+
+             // Return the Grid as the root of the DataTemplate
```
Comment (Chrispineagunja) [inline MAUI/NumericEntry/UpDown-Button.md:196]: // Return the Grid as the root of the DataTemplate.

**Review by Chrispineagunja — COMMENTED** [review]
Content review completed.

**Pull request merged**